pkg/cmd/cli/cmd/deploy.go

```diff
@@ -211,6 +211,9 @@ func (o DeployOptions) RunDeploy() error {
 // deploy launches a new deployment unless there's already a deployment
 // process in progress for config.
 func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot deploy a paused deploymentconfig")
```

Comment:
I think "deploymentconfig" is fine as we already return it for anything cli-related. Try creating, or editing a dc and you will get "deploymentconfig" back. Cli users that pause their configs are probably familiar with our api too (/deploymentconfigs/namespace/name)